.gitignore

```diff
@@ -26,6 +26,6 @@ doctemp/*
 source/obj
 source/Irrlicht/libIrrlichtServer.a
 # stuff output by examples
-examples_tests/*/bin/*
+examples_tests/*
```

Comment:
shouldn't even be here anymore

Reply:
no `examples_tests` path should be here

should be in the submodule's gitignore